apis/python/src/tiledbsoma/io/shaping.py

```diff
@@ -329,7 +296,13 @@ def _treewalk(
     *,
     node_name: str | None = None,
     visitor: Callable[..., bool],
-    args: SizingArgs,
+    nobs: int | None,
+    nvars: Dict[str, int] | None,
+    ms_name: str | None,
+    coll_name: str | None,
+    verbose: bool,
+    check_only: bool,
+    output_handle: Printable | None,
```

Comment:
```suggestion
    **kwargs
```
The `_treewalk` and `_recurse` methods and the `visitor` call below can/should just forward whatever parameters are passed into the method.

Reply:
Thanks! Another commit coming up.